app/views/feeds/_form_auth_token.html.erb

```diff
@@ -1,15 +1,25 @@
-<div class="row">
-  <div class="col mb-4">
-    <div class="form-floating">
-      <%= form.text_field :label %>
-      <%= form.label :label %>
+<div class="mb-4 prx-field-group">
+  <div class="row">
+    <div class="col-6">
+      <div class="form-floating">
+        <%= form.text_field :label %>
+        <%= form.label :label %>
+      </div>
     </div>
-  </div>
 
-  <div class="col mb-4">
-    <div class="form-floating">
-      <%= form.text_field :token %>
-      <%= form.label :token %>
-    </div>
+    <% unless form.object.new_record? %>
+      <div class="col-6">
+        <div class="form-floating">
+          <%= form.text_field :token %>
+          <%= form.label :token %>
+        </div>
+      </div>
+    <% end %>
   </div>
+
+  <% unless form.object.new_record? %>
+    <%= link_to podcast_feed_path({podcast_id: @podcast, id: feed}, feed_tokens_destroy_params(form)), class: "btn" do %>
```

Comment:
So this works, but it does generate a very odd looking href:

```
/podcasts/id=680&podcast_id=815/feeds/680?feed%5Bfeed_tokens_attributes%5D%5B0%5D%5B_destroy%5D=1&feed%5Bfeed_tokens_attributes%5D%5B0%5D%5Bid%5D=70
```

I'm actually a bit surprised that works!

But what you actually want is to just pass the 2 models in-order to the path helper, with the query params hash as the last argument:

```ruby
podcast_feed_path(feed.podcast, feed, feed_tokens_destroy_params(form))
```

(Also changed out the `@podcast` there, as it's a _bit_ of a rails convention to only reference those `@` vars in the top-level view).